ndcube/cube_utils.py

```diff
@@ -183,9 +184,9 @@ def index_sequence_as_cube(cubesequence, item):
     if item is None or (isinstance(item, tuple) and None in item):
         raise IndexError("None indices not supported")
     try:
```

Comment:
A better approach would be to just check the first item i.e item[0] of tuple should be slice.
So something like - 
```
if isinstance(item_tuple, tuple):
     if isinstance(item_tuple[0], slice):
        return get_cube_from_sequence(cubesequence, item_tuple, type_slice='sequence_as_cube')

return get_cube_from_sequence(cubesequence, item_tuple)

Reply:
This will remove the try and catch as it is vague in telling us which part of the code has failed.

Reply:
Hi @abit2.  Thanks for taking a look at this.  To be honest, I don't fully understand the difference of what ```slice_type=None``` and ```slice_type='sequence_as_cube``` mean.  Can you explain it to me?

Reply:
https://github.com/sunpy/ndcube/pull/16 
so it needed some cleanup that's all :)
both of them were trying to do the same thing.